README.rst

```diff
@@ -0,0 +1,591 @@
+tractor
+=======
+A minimalist `actor model`_ built on multiprocessing_ and trio_.
+
+|travis|
+
+.. |travis| image:: https://img.shields.io/travis/tgoodlet/tractor/master.svg
+    :target: https://travis-ci.org/tgoodlet/tractor
+
+``tractor`` is an attempt to take trionic_ concurrency concepts and apply
+them to distributed-multicore Python.
+
+``tractor`` lets you run and spawn Python *actors*: separate processes which are internally
+running a ``trio`` scheduler and task tree (also known as an `async sandwich`_).
+
+Actors communicate with each other by sending *messages* over channels_, but the details of this
+in ``tractor`` is by default hidden and *actors* can instead easily invoke remote asynchronous
+functions using *portals*.
+
+``tractor``'s tenets non-comprehensively include:
+
+- no spawning of processes *willy-nilly*; causality_ is paramount!
+- `shared nothing architecture`_
+- remote errors `always propagate`_ back to the caller
+- verbatim support for ``trio``'s cancellation_ system
+- no use of *proxy* objects to wrap RPC calls
+- an immersive debugging experience
+- be simple, be small
```

Comment:
Don't bother with this last point, its a sematics game. Small or large are *preceptions* based on perceived feature breadth and depth and fitness to the problem at hand.

I think its more interesting to instead really define your scope and list antigoals. These are things that help keep focus and a well focused code base will never feel large and boated.

Reply:
Swap with *first class `pypy3.6` support* (once it comes out of course)?